sig-feature-store/README.md

```diff
@@ -0,0 +1,52 @@
+<!---
+This is an autogenerated file!
+
+Please do not edit this file directly, but instead make changes to the
+sigs.yaml file in the project root.
+
+To understand how this file is generated, see https://github.com/kubeflow/community/generator/README.md
+--->
+# Feature Store Special Interest Group
+
+SIG Feature Store covers the definition, management, storage, discovery, and serving of features to models.
+
+The [charter](charter.md) defines the scope and governance of the Feature Store Special Interest Group.
+
+## Meetings
+* Monthly SIG Meeting: [Wednesdays at 18:00 PT (Pacific Time)]() (monthly). [Convert to your timezone](http://www.thetimezoneconverter.com/?t=18:00&tz=PT%20%28Pacific%20Time%29).
+  * [Meeting notes and Agenda](https://docs.google.com/document/d/1h2BMuN6V6cGFQrSRk7msmYVmZdrouLRV_EPR_tphZrU/edit#).
+
+## Leadership
+
+### Chairs
+The Chairs of the SIG run operations and processes governing the SIG.
+
+* Kevin Stumpf (**[@kevinstumpf](https://github.com/kevinstumpf)**)
+* Oleksii Moskalenko (**[@pyalex](https://github.com/pyalex)**), Gojek
+* Willem Pienaar (**[@woop](https://github.com/woop)**), Gojek
+
+### Technical Leads
+The Technical Leads of the SIG establish new subprojects, decommission existing
+subprojects, and resolve cross-subproject technical issues and decisions.
+
+* Kevin Stumpf (**[@kevinstumpf](https://github.com/kevinstumpf)**)
+* Oleksii Moskalenko (**[@pyalex](https://github.com/pyalex)**), Gojek
+* Willem Pienaar (**[@woop](https://github.com/woop)**), Gojek
+
+## Contact
+- Slack: [#feast](https://kubeflow.slack.com/messages/feast)
+- [Mailing list](https://groups.google.com/forum/#!forum/kubeflow-discuss)
+- [Open Community Issues/PRs](https://github.com/kubeflow/community/labels/sig%2Farea/sig-feature-store)
+- GitHub Teams:
+    - [@kubeflow/sig-feature-store](https://github.com/orgs/kubeflow/teams/sig-feature-store) - Feature store development and roadmap
+
+## Subprojects
+
+The following subproject(s) are owned by sig-area/sig-feature-store:
+### feast
+- **Owners:**
+  - https://github.com/feast-dev/feast/blob/master/OWNERS
```

Comment:
Will feast donate to Kubeflow?

Reply:
It's a bit overloaded as is because we are hosting our #feast channel in Kubeflow.org Slack. I wanted to make sure that there is a group taking responsibility of the project from the Kubeflow perspective, but mostly in terms of integration.

There wouldn't be a contribution/donation into the Kubeflow org.

@jlewi @gaocegege should we remove that line?

Reply:
I am pretty sure @jlewi said here: https://github.com/kubeflow/community/pull/408#issuecomment-692396749 that SIG cant have assets they own. (And that makes sense)

Reply:
I think it may be confusing for readers since the project is not owned by Kubeflow. But I have no objection to it.

Reply:
I am saying to remove it, because it a SIG cant have a sub-project?

Reply:
Oh SGTM.

Reply:
@woop ?

Reply:
@thesuperzapper @gaocegege sgtm, removed!